advisor/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
Commit message nits: A "module" is in IntelliJ what a "project" is in Gradle, but what I believe you mean here is "class" (in the title) and "implementations" (in the body).